dev-python/flask-cors/flask-cors-3.0.8.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7,8} )
+DISTUTILS_USE_SETUPTOOLS=bdepend
+
+inherit distutils-r1
+
+if [[ "${PV}" == "9999" ]]; then
+	EGIT_REPO_URI="https://github.com/wcdolphin/${PN}.git"
+	inherit git-r3
+else
+	MY_PN="Flask-Cors"
+	MY_P="${MY_PN}-${PV}"
+	SRC_URI="mirror://pypi/${MY_P:0:1}/${MY_PN}/${MY_P}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+DESCRIPTION="A Flask extension for Cross Origin Resource Sharing (CORS)"
+HOMEPAGE="https://github.com/wcdolphin/flask-cors https://pypi.org/project/Flask-Cors/"
+
+LICENSE="MIT"
+SLOT="0"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-python/flask[${PYTHON_USEDEP}]
+	dev-python/six[${PYTHON_USEDEP}]
+"
+DEPEND="${RDEPEND}"
+
+S="${WORKDIR}/${MY_P}"
+
+distutils_enable_tests nose
+distutils_enable_sphinx doc --no-autodoc
+
+python_prepare_all() {
+	distutils-r1_python_prepare_all
+	# test is failing with >flask-1.1, see #253 on gh
+	rm tests/decorator/test_exception_interception.py || die
+}
+
+python_test() {
+	nosetests -vv || die "Testing failed with ${EPYTHON}"
+}
```

Comment:
The default `distutils_enable_tests nose` should work without this defined at all.

Reply:
Yes you are right